Config.uk

```diff
@@ -35,10 +35,17 @@ config LIBEMBEDDEDGCOV_OUTPUT_SERIAL_HEXDUMP
 	bool "Console"
 config LIBEMBEDDEDGCOV_OUTPUT_BINARY_FILE
 	bool "File"
+config LIBEMBEDDEDGCOV_OUTPUT_BINARY_MEMORY
+	bool "Memory"
 endchoice
 
 config LIBEMBEDDEDGCOV_OUTPUT_BINARY_FILENAME
 	depends on LIBEMBEDDEDGCOV_OUTPUT_BINARY_FILE
 	string "Filename for coverage output"
 
+config LIBEMBEDDEDGCOV_OUTPUT_BINARY_MEMORY_ADDRESS
+	depends on LIBEMBEDDEDGCOV_OUTPUT_BINARY_MEMORY
+	hex "Address for coverage output"
+	default 0x42000000
```

Comment:
This implies that the memory is mapped already. Perhaps you can add a `help` section here to provide that information, and also update the `README` accordingly. Another problem is that there are no guarantees that there will be enough space in the buffer. That is a problem of embedded-gcov, so one option is to create an additional feature that (re)allocates memory dynamically instead.

Reply:
Modified README and help sections. Need to provide a patch for memory (re)allocation now.